example/pandas/dataframe.py

```diff
@@ -37,3 +37,22 @@
     df = c.query_dataframe('SELECT * FROM table(test)')
     print(df)
     print(df.describe())
+
+    # Converting query results to a variety of formats with dbapi
+    with connect('proton://localhost') as conn:
```

Comment:
I saw in README there is a section about client dataframe methods. Could you add some doc about dbapi methods there?